frontends/web/src/routes/account/account.tsx

```diff
@@ -131,7 +131,10 @@ export function Account({
     return () => unsubscribe(unsubscribeList);
   }, [code, onAccountChanged, onStatusChanged, status]);
 
-  const moonpayBuySupported = useLoad(isMoonpayBuySupported(code));
+  useEffect(() => {
+    getExchangeBuySupported(code)()
+      .then(supported => setExchangeBuySupported(supported.exchanges.length > 0));
+  }, [code]);
```

Comment:
alternative the useLoad hook should update with a second arg that takes a dependency list and should (🤞 ) update if the dep changes:

https://github.com/digitalbitbox/bitbox-wallet-app/blob/f4e5a031d6b873821586142d9f931b1eecb5e9d0/frontends/web/src/routes/device/bitbox02/settings.tsx#L53

Reply:
TIL :D Thanks